AppKit/CPImage.j

```diff
@@ -208,6 +208,17 @@ function CPAppKitImage(aFilename, aSize)
     return self;
 }
 
+/*!
+    Initializes and returns an image object with the contents of the specified URL.
+    @param url the URL representing the image
+    @return the initialized image
+*/
+- (id)initWithContentsOfURL:(CPURL)url
+{
+    return [self initWithContentsOfFile: [url absoluteString]];
```

Comment:
no whitespace after colon

Reply:
Removed the space and committed the fix to the branch.